substrate/client/cli/src/params/import_params.rs

```diff
@@ -80,6 +80,38 @@ pub struct ImportParams {
 	/// Providing `0` will disable the cache.
 	#[arg(long, value_name = "Bytes", default_value_t = 1024 * 1024 * 1024)]
 	pub trie_cache_size: usize,
+
+	/// Warm up the trie cache.
+	///
+	/// No warmup if flag is not present. Using flag without value chooses non-blocking warmup.
+	#[arg(long, value_name = "STRATEGY", value_enum, num_args = 0..=1, default_missing_value = "non-blocking")]
```

Comment:
I guess it makes sense to have warm up on by default. But people could run into memory issues maybe and need to up `trie_cache_size`?

Reply:
I'm not sure, in the beginning we only thought about warmup for parachain nodes. Why do you think we need it by default?

Reply:
that allows us to not pass the value to --warm-up-trie-cache. So four options are possible:
[no flag]: no warm-up
--warm-up-trie-cache: non-blocking warmup
--warm-up-trie-cache non-blocking: same as just the flag, non-blocking warmup
--warm-up-trie-cache blocking: blocking warmup

Reply:
Aye. I misunderstand the attribute. Its not the default value but if it is set without explicitly specifying.